adapters/adnuntius/adnuntiustest/supplemental/check-dsa-error.json

```diff
@@ -0,0 +1,46 @@
+{
+	"mockBidRequest": {
+		"id": "test-request-id",
+		"user": {
+			"id": "1kjh3429kjh295jkl"
+		},
+		"site": {
+			"ext":{
+				"data" : {
+					"key": ["value"]
+				}
+			}
+		},
+		"regs": {
+			"ext": ""
+		},
+		"imp": [
+			{
+				"id": "test-imp-id",
+				"banner": {
+					"format": [
+						{
+							"w": 300,
+							"h": 250
+						},
+						{
+							"w": 300,
+							"h": 600
+						}
+					]
+				},
+				"ext": {
+					"bidder": {
+						"auId": "123"
+					}
+				}
+			}
+		]
+	},
+	"expectedMakeRequestsErrors": [
+		{
+			"value": "failed to parse URL: [failed to parse Adnuntius endpoint: failed to parse ExtRegs in Adnuntius GDPR check: json: cannot unmarshal string into Go value of type openrtb_ext.ExtRegs]",
```

Comment:
I know this is not new logic but I just want to point out that this error is a little misleading since the error does not appear to be due to parsing a URL or the endpoint but rather due to a failure parsing the ExtRegs while attempting to generate the URL. Perhaps you want to address this in a future PR?